inst/pages/97_extra_materials.qmd

```diff
@@ -294,7 +294,7 @@ var_explained <- e/sum(e[e>0])*100
 This section has a cheatsheet for translating common functions in phyloseq
 to TreeSE/mia with example code.
 
-```{r}
+```{r,message=FALSE, warning=FALSE, eval=FALSE}
```

Comment:
add space in "r, message" here and elsewhere

Reply:
In fact, on the same go you could move the messages, warning and eval into quarto header in chunks of this PR

Like 
#| message: false
#| warning: false
#| eval: false

Reply:
Should be better now (couldn't be able to do the way you said but pretty much the same I believe)